pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -26,7 +26,8 @@ static __always_inline void update_conn_state(conn_tuple_t *t, conn_stats_ts_t *
     }
 }
 
-static __always_inline void update_conn_stats(conn_tuple_t *t, size_t sent_bytes, size_t recv_bytes, u64 ts, conn_direction_t dir) {
+static __always_inline void update_conn_stats(conn_tuple_t *t, size_t sent_bytes, size_t recv_bytes, u64 ts, conn_direction_t dir,
+                                              __u32 segs_in, __u32 segs_out, segment_count_increment_t segs_type) {
```

Comment:
Could you reorder the parameters to be `segs_out` and then `segs_in`? That would be consistent with the sent, recv order earlier in the signature.